packages/ai/src/Button.ts

```diff
@@ -0,0 +1,16 @@
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+/**
+ * @class
+ * @constructor
+ * @extends UI5Element
+ */
+@customElement({
+	tag: "ui5-ai-button",
+})
+class Button extends UI5Element {
```

Comment:
I think this is the better export, because `AIButton` would duplicate the `-ai` from the library name.